diff -urB dgamelaunch.bak/examples/dgamelaunch.conf dgamelaunch.new/examples/dgamelaunch.conf
--- dgamelaunch.bak/examples/dgamelaunch.conf	2020-03-07 06:27:35.000000000 -0500
+++ dgamelaunch.new/examples/dgamelaunch.conf	2026-05-23 03:21:19.009017983 -0400
@@ -1,12 +1,7 @@
-# This is a sample dgamelaunch configuration file. Only bash-style comments
-# are allowed, such as this. Each configuration option will be explained
-# along with its default value.
-
-
-
-# Global config variables:
-
+# This is a sample dgamelaunch configuration file.
+# Also, functional.
 
+### Global config variables:
 # Max amount of registered users to allow. Has no effect if dgl was
 # compiled with SQLite
 maxusers = 64000
@@ -41,20 +36,20 @@
 #                 ]
 
 # Path to a prepared chroot jail.
-chroot_path = "/opt/nethack/nethack.alt.org/"
+chroot_path = "/opt/dgamelaunch"
 
 # From inside the jail, dgamelaunch's working directory for rcfiles/ttyrec/etc
-dglroot = "/dgldir/"
+dglroot = "/var/"
 
 # Server ID string
-server_id = "nethack.alt.org - http://nethack.alt.org/"
+server_id = "Generic NetHack-Enabled dgamelaunch Server"
 
 # From inside the jail, location of a banner file, the topmost line will be
 # shown in submenus that cannot be defined separately.
 # Some string substitution is done for the file:
 # $VERSION   replaced with "dgamelaunch v" + dgl version number.
 # $SERVERID  replaced with the server_id string, as defined above.
-banner = "/dgl-banner"
+banner = "/etc/menu/dgl-banner"
 
 # The following two options are fairly insecure. They will force us to
 # load the password/group database into memory while still having root
@@ -152,7 +147,7 @@
 
 # First, the menu shown to anonymous user:
 menu["mainmenu_anon"] {
-        bannerfile = "dgl_menu_main_anon.txt"
+        bannerfile = "/etc/menu/dgl_menu_main_anon.txt"
         cursor = (5,18)
         commands["l"] = ask_login
         commands["r"] = ask_register
@@ -165,7 +160,7 @@
 menu["mainmenu_user"] {
 # contents of this file are written to screen.
 # the file must be inside the chroot.
-        bannerfile = "dgl_menu_main_user.txt"
+        bannerfile = "/etc/menu/dgl_menu_main_user.txt"
 # after which cursor is moved to this location
 # if cursor-definition is missing, the cursor is put
 # to the end of the last line of the banner.
@@ -176,15 +171,18 @@
         commands["c"] = chpasswd
         commands["e"] = chmail
         commands["w"] = watch_menu
-        commands["o"] = ifnxcp "/dgl-default-rcfile.nh343" "%ruserdata/%n/%n.nh343rc",
-		        exec "/bin/virus" "%ruserdata/%n/%n.nh343rc"
-        commands["p"] = play_game "NH343"
+        commands["o"] = ifnxcp "/etc/dgl-default-rcfile.nh500" "%ruserdata/%n/%n.nh500rc",
+		        exec "/bin/virus" "%ruserdata/%n/%n.nh500rc"
+        commands["O"] = ifnxcp "/etc/dgl-default-rcfile.nh500" "%ruserdata/%n/%n.nh500rc",
+		        exec "/bin/nano" "%ruserdata/%n/%n.nh500rc"
+        commands["p"] = play_game "nh500curses"
+	commands["P"] = play_game "nh500tty"
         commands["q"] = quit
 }
 
 # this menu is shown when user presses '?' in the watching menu
 menu["watchmenu_help"] {
-	bannerfile = "dgl_menu_watchmenu_help.txt"
+	bannerfile = "/etc/menu/dgl_menu_watchmenu_help.txt"
 	commands["qQ "] = return
 }
 
@@ -193,29 +191,41 @@
 
 # Next, we'll define one game's data:
 
-#DEFINE {
+DEFINE {
 #  # From inside the jail, the location of the binary to be launched.
-#  game_path = "/bin/nethackstub"
-#
+  game_path = "/bin/nh500"
+
+#  # If the game uses an ancient encoding, you may specify "ibm" or "dec".
+#  # If set to "ask", the game will be run with --print-charset beforehand,
+#  # expected to return one of these values.
+#  encoding = "unicode"
+
 #  # Full name of the game
-#  game_name = "NetHack stub"
-#
+  game_name = "NetHack 5.0.0 (tty)"
+
 #  # Short name, used in the watching menu
-#  short_name = "NHstb"
-#
+  short_name = "nh500tty"
+
 #  # arguments for when we exec the binary
-#  game_args = "/bin/nethackstub",
-#	      "foo",
-#	      "user:%n",
-#	      "shed_uid:%u",
-#	      "bar"
-#
+  game_args = "/bin/nethackstub", "-u", "%n", "-wtty"
+
 #  # From inside the jail, where dgamelaunch should put mail.
-#  spooldir = "/var/mail/"
-#
+  spooldir = "/mail/"
+
 #  # From inside the jail, the default .nethackrc that is copied for new users.
-#  # rc_template = "/dgl-default-rcfile"
-#
+  rc_template = "/etc/dgl-default-rcfile.nh500"
+# Format for writing users' own RC files
+  rc_fmt = "%ruserdata/%n/%n.nh500rc"
+
+#  # Make sure the inprogress dir actually exists. default is "inprogress/"
+#  # Each game you define here must have it's own.
+  inprogressdir = "%rinprogress-nh500/"
+
+# The place where ttyrecs are stored for this game.
+# If this is not defined, ttyrecs are not saved for this game.
+# Leaving this undefined also means the games cannot be spectated.
+  ttyrecdir = "%ruserdata/%n/ttyrec/"
+
 #  # If player idles longer than max_idle_time seconds, the game will
 #  # receive a sighup. Default value is 0, which disables the idling timer.
 #  max_idle_time = 2000
@@ -229,57 +239,74 @@
 #  # will be sorted to appear before lower weights.
 #  # 
 #  extra_info_file = "%rgamedir/%n.extrainfo"
-#
-#  # Make sure the inprogress dir actually exists. default is "inprogress/"
-#  # Each game you define here must have it's own.
-#  inprogressdir = "%rinprogress-nethackstub/"
-#
+
 #  # We can also define per-game commands, that are executed
 #  # when the game starts:
 #  # commands = chdir "/dgldir", mkdir "foo_%u_%g"
-#
-#  # If the game uses an ancient encoding, you may specify "ibm" or "dec".
-#  # If set to "ask", the game will be run with --print-charset beforehand,
-#  # expected to return one of these values.
-#  encoding = "unicode"
-#}
+  # back up savefile
+  commands = cp "/var/nh500/save/%u%n.gz" "/var/nh500/save/%u%n.gz.bak",
+  # set NETHACKOPTIONS to point to the rcfile
+             setenv "NETHACKOPTIONS" "@%ruserdata/%n/%n.nh500rc",
+  # set up nethack mail stuff, assuming it's compiled with it...
+             setenv "MAIL" "/mail/%n",
+             setenv "SIMPLEMAIL" "1",
+  # don't let the mail file grow
+             unlink "/mail/%n"
 
+}
+# Defining it again for the curses option.
+DEFINE {
+  game_path = "/bin/nh500"
+  game_name = "NetHack 5.0.0 (curses)"
+  short_name = "nh500curses"
+  game_args = "/bin/nethackstub", "-u", "%n", "-wcurses"
+  spooldir = "/mail/"
+  rc_template = "/etc/dgl-default-rcfile.nh500"
+  rc_fmt = "%ruserdata/%n/%n.nh500rc"
+  inprogressdir = "%rinprogress-nh500/"
+  ttyrecdir = "%ruserdata/%n/ttyrec/"
+  commands = cp "/var/nh500/save/%u%n.gz" "/var/nh500/save/%u%n.gz.bak",
+             setenv "NETHACKOPTIONS" "@%ruserdata/%n/%n.nh500rc",
+             setenv "MAIL" "/mail/%n",
+             setenv "SIMPLEMAIL" "1",
+             unlink "/mail/%n"
+}
 
 #
 # the second game
 #
 #
 
-DEFINE {
-  game_path = "/nh343/nethack"
-  game_name = "NetHack 3.4.3"
-  short_name = "NH343"
-
-  game_args = "/nh343/nethack", "-u", "%n"
-
-  spooldir = "/mail/"
-  rc_template = "/dgl-default-rcfile.nh343"
-
-  rc_fmt = "%ruserdata/%n/%n.nh343rc"
-
-  inprogressdir = "%rinprogress-nh343/"
-
+#DEFINE {
+#  game_path = "/nh343/nethack"
+#  game_name = "NetHack 3.4.3"
+#  short_name = "NH343"
+#
+#  game_args = "/nh343/nethack", "-u", "%n"
+#
+#  spooldir = "/mail/"
+#  rc_template = "/dgl-default-rcfile.nh343"
+#
+#  rc_fmt = "%ruserdata/%n/%n.nh343rc"
+#
+#  inprogressdir = "%rinprogress-nh343/"
+#
 # The place where ttyrecs are stored for this game.
 # If this is not defined, ttyrecs are not saved for this game.
 # Leaving this undefined also means the games cannot be spectated.
-  ttyrecdir = "%ruserdata/%n/ttyrec/"
-
-
+#  ttyrecdir = "%ruserdata/%n/ttyrec/"
+#
+#
   # back up savefile
-  commands = cp "/nh343/var/save/%u%n.gz" "/nh343/var/save/%u%n.gz.bak",
+#  commands = cp "/nh343/var/save/%u%n.gz" "/nh343/var/save/%u%n.gz.bak",
   # set NETHACKOPTIONS to point to the rcfile
-             setenv "NETHACKOPTIONS" "@%ruserdata/%n/%n.nh343rc",
+#             setenv "NETHACKOPTIONS" "@%ruserdata/%n/%n.nh343rc",
   # set up nethack mail stuff, assuming it's compiled with it...
-             setenv "MAIL" "/mail/%n",
-             setenv "SIMPLEMAIL" "1",
+#             setenv "MAIL" "/mail/%n",
+#             setenv "SIMPLEMAIL" "1",
   # don't let the mail file grow
-             unlink "/mail/%n"
-}
+#             unlink "/mail/%n"
+#}
 
 #
 # third game
diff -urB dgamelaunch.bak/examples/dgl_menu_main_user.txt dgamelaunch.new/examples/dgl_menu_main_user.txt
--- dgamelaunch.bak/examples/dgl_menu_main_user.txt	2020-03-07 06:27:35.000000000 -0500
+++ dgamelaunch.new/examples/dgl_menu_main_user.txt	2026-05-23 03:21:44.601752899 -0400
@@ -11,8 +11,10 @@
  c) Change password
  e) Change email address
  w) Watch games in progress
- o) Edit options
- p) Play NetHack 3.4.3
+ o) Edit options (Virus)
+ O) Edit options (Nano)
+ p) Play NetHack 5.0.0 (curses)
+ P) Play NetHack 5.0.0 (tty)
  q) Quit
 
  =>
